Paralayout/UIViewAdditions_Alignment.swift

```diff
@@ -242,9 +176,14 @@ extension UIView {
             assertionFailure("Can't align view without a superview!")
             return
         }
-        
+
+        // Resolve the position before aligning, since we always want to use the top left corner (i.e. the origin) of
+        // superview, regardless of the layout direction. Without this, we'll hit the mismatched alignment positions
+        // alert when using a leading/trailing position.
+        let resolvedPosition = ResolvedPosition(resolving: position, with: effectiveUserInterfaceLayoutDirection)
```

Comment:
This could be refactored to be a bit more efficient, since right now it jumps back and forth between `Position` and `ResolvedPosition`. I'm tempted to land this as-is and do that later, though, since I think we'll end up refactoring this class a fair bit for #49 anyway.